src/gamma_inc.jl

```diff
@@ -0,0 +1,570 @@
+    elseif x > -0.227
+        return _log1pmx_ker(x)
+    elseif x > -0.4
+        u = (x+0.25)/0.75
+        return _log1pmx_ker(u) - 3.76820724517809274e-2 + 0.25*u
+    elseif x > -0.6
+        u = (x+0.5)*2.0
+        return _log1pmx_ker(u) - 1.93147180559945309e-1 + 0.5*u
+    else
+        u = (x+0.625)/0.375
+        return _log1pmx_ker(u) - 3.55829253011726237e-1 + 0.625*u
+    end
+end
+"""
+    logmxp1(x::Float64)
+Return `log(x) - x + 1` carefully evaluated.
+"""
+function logmxp1(x::Float64)
+    if x <= 0.3
+        return (log(x) + 1.0) - x
+    elseif x <= 0.4
+        u = (x-0.375)/0.375
+        return _log1pmx_ker(u) - 3.55829253011726237e-1 + 0.625*u
+    elseif x <= 0.6
+        u = 2.0*(x-0.5)
+        return _log1pmx_ker(u) - 1.93147180559945309e-1 + 0.5*u
+    else
+        return log1pmx(x - 1.0)
+    end
+end
+
+"""
+   rgamma1pm1(a)
+
+   Computation of 1/Gamma(a+1) - 1 for -0.5<=a<=1.5 : ``1/\\Gamma (a+1) - 1``
+   Uses the relation gamma(a+1) = a*gamma(a)
+"""
+function rgamma1pm1(a::Float64)
+    t=a
+    rangereduce = a > 0.5
+    t = rangereduce ? a-1 : a #-0.5<= t <= 0.5
+    if t == 0.0
+        return 0.0
+    elseif t < 0.0        
+        top = @horner(t , -.422784335098468E+00 , -.771330383816272E+00 , -.244757765222226E+00 , .118378989872749E+00 , .930357293360349E-03 , -.118290993445146E-01 , .223047661158249E-02 , .266505979058923E-03 , -.132674909766242E-03)
+        bot = @horner(t , 1.0 , .273076135303957E+00 , .559398236957378E-01)
+        w = top/bot
+        return rangereduce ? t*w/a : a*(w+1)
+    else
+        top = @horner(t , .577215664901533E+00 , -.409078193005776E+00 , -.230975380857675E+00 , .597275330452234E-01 , .766968181649490E-02 , -.514889771323592E-02 , .589597428611429E-03)
+        bot = @horner(t , 1.0 , .427569613095214E+00 , .158451672430138E+00 , .261132021441447E-01 , .423244297896961E-02) 
+        w = top/bot
+        return rangereduce ? (t/a)*(w - 1.0) : a*w
+    end
+end
+"""
+    rgammax(a,x)
+
+Evaluation of exp(-x)*x^a/gamma(a) : ``1/\\Gamma(a) e^{-x} x^{a}``
+"""
+function rgammax(a::Float64,x::Float64)
+    if x == 0.0
+        return 0.0
+    elseif a >= 20.0
+        u =x/a
+        if u == 0.0
+            return 0.0
+        end
+        t = (1.0/a)^2
+        t1 = (((0.75*t - 1.0)*t + 3.5)*t - 105.0)/(a*1260.0) #Using Stirling Series : https://dlmf.nist.gov/5.11.1
+        t1 = t1 + a*logmxp1(u)
+        if t1 >= exparg
+            return rt2pin*sqrt(a)*exp(t1)
+        end
+    else
+        t = a*log(x) - x
+        if t < exparg
+            return 0.0
+        elseif a >= 1.0
+            return exp(t)/gamma(a)
+        else
+            return (a*exp(t))*(1.0 + rgamma1pm1(a))
+        end
+    end
+end   
+"""
+    gamma_p_cf(a, x, ind)
+
+Computes P(a,x) by continued fraction expansion given by : ``\\frac{1}{1-\\frac{z}{a+1+\\frac{z}{a+2-\\frac{(a+1)z}{a+3+\\frac{2z}{a+4-\\frac{(a+2)z}{a+5+\\frac{3z}{a+6-\\dots}}}}}}}``.
```

Comment:
use the block math syntax: ```math

Reply:
Should this be done for all the equations in docstrings? Or this particular one?
And I didn't get why did you ask that? Won't ` ``equation`` ` suffice?

Reply:
Done for this particular one. Since it was big I guess.